src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -323,6 +320,27 @@ public FluentDataGrid(LibraryConfiguration configuration) : base(configuration)
     [Parameter]
     public RenderFragment? LoadingContent { get; set; }
 
+    /// <summary>
+    /// Gets or sets the callback that is invoked when the asynchronous loading state of items changes and <see cref="IAsyncQueryExecutor"/> is used.
+    /// </summary>
+    /// <remarks>The callback receives a <see langword="true"/> value when items start loading
+    /// and a <see langword="false"/> value when the loading process completes.</remarks>
+    [ExcludeFromCodeCoverage(Justification = "This method requires a db connection and is to complex to be tested with bUnit.")]
```

Comment:
You can simulate a DB using a method including a `Delay`. In a new PR?

Reply:
Will try!